core/src/main/java/com/novoda/merlin/Merlin.java

```diff
@@ -18,10 +16,6 @@
         this.registerer = registerer;
     }
 
-    public void setEndpoint(String endpoint) {
```

Comment:
This breaks a public API. While it is recommended to use the Builder, we have not enforced it.  This needs some explanation in the README how to migrate.